src/Numpy/Numpy.csproj

```diff
@@ -34,9 +34,16 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="BH.UI.Python.Included" Version="3.7.3.9-alpha" />
-    <PackageReference Include="BH.UI.Python.Runtime" Version="3.7.4-alpha" />
     <PackageReference Include="Microsoft.CSharp" Version="4.6.0" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Reference Include="Python.Runtime">
+      <HintPath>..\..\..\Python_Toolkit\Build\Python.Runtime.dll</HintPath>
+    </Reference>
+    <Reference Include="Python_Engine">
+      <HintPath>..\..\..\Python_Toolkit\Build\Python_Engine.dll</HintPath>

```

Comment:
This wasn't working for me - but if I removed `Installing keras in python at compile time` then it did. With that in, it had a 9009 error, but removing that would make it build (but Python did not install - more on that in Python_Toolkit).